package.json

```diff
@@ -34,7 +34,9 @@
     "pretest": "npm run build:dll",
     "test": "cross-env NODE_ENV=test karma start",
     "test:watch": "npm run test --silent -- --watch",
-    "tsd": "cpx './src/**/*.d.ts' dist/commonjs"
+    "tsd": "cpx './src/**/*.d.ts' dist/commonjs",
+    "tsd:lint": "tslint './src/**/*.d.ts'",
+    "tsd:lint:fix": "npm run tsd:lint -- --fix"
```

Comment:
Let's `silence` the extra NPM error blocks this would produce on a failure:

```diff
-npm run tsd:lint -- --fix
+npm run -s tsd:lint -- --fix
```